touch_element/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+idf_build_get_property(target IDF_TARGET)
+
+if(${target} STREQUAL "linux")
+    return() # This component is not supported by the POSIX/Linux simulator
+endif()
+
+set(TOUCH_ELEMENT_COMPATIBLE_TARGETS "esp32s2" "esp32s3")
```

Comment:
no need this check in the cmake file  if we specify the "targets" in the idf_component.yml file: https://docs.espressif.com/projects/idf-component-manager/en/latest/reference/manifest_file.html#targets